public/css/dashboard.css

```diff
@@ -465,20 +477,17 @@ body {
         grid-row: span 2;
     }
 
+    body.no-title .user-menu {
```

Comment:
Does this work on smaller screens? I think it should not be fixed on smaller screens right?

Reply:
It is now fixed only for no-title screens above 800px width.
I don't find a way to display it above the map with the grid system